action_files/test_models/src/data.py

```diff
@@ -42,24 +42,16 @@ def get_data():
     ]
 
     Y_df, S_df, tags = aggregate(Y_df, spec)
-    Y_df = Y_df.reset_index()
 
     # Train/Test Splits
     Y_test_df = Y_df.groupby('unique_id').tail(8)
     Y_train_df = Y_df.drop(Y_test_df.index)
 
-    Y_test_df = Y_test_df.set_index('unique_id')
-    Y_train_df = Y_train_df.set_index('unique_id')
-
-    sf = StatsForecast(df=Y_train_df,
-                       models=[ETS(season_length=4, model='ZZA')],
+    sf = StatsForecast(models=[AutoETS(season_length=4, model='ZZA')],
                        freq='QS', n_jobs=-1)
-    Y_hat_df = sf.forecast(h=8, fitted=True)
-    Y_fitted_df = sf.forecast_fitted_values()
+    Y_hat_df = sf.forecast(df=Y_train_df, h=8, fitted=True).reset_index()
```

Comment:
I recommend instead setting the `NIXTLA_ID_AS_COL` env variable. Can be either in the CI job or in this script. That way you won't have to reset the index anywhere.

Reply:
Good suggestion